ext/CreateParametersExt.jl

```diff
@@ -51,4 +54,52 @@ function LandParameters(toml_dict::CP.AbstractTOMLDict)
     )
 end
 
+# TODO: Should this be organized into multiple files?
```

Comment:
would be good to remove this TODO before merging! I think it's okay to keep everything in this file but you have a better vision of how large this might become so whatever you think is best works for me

Reply:
We can always split it up easily later if needed, so I will just remove this for now.